classes/AddressFormat.php

```diff
@@ -515,7 +515,7 @@ class_exists($nameObject)) {
     public static function getOrderedAddressFields($idCountry = 0, $splitAll = false, $cleaned = false)
     {
         $out = array();
-        $fieldSet = explode(PHP_EOL, AddressFormat::getAddressCountryFormat($idCountry));
+        $fieldSet = explode("\n", AddressFormat::getAddressCountryFormat($idCountry));
```

Comment:
Because, I just learn this format value is stored in database
```suggestion
        $fieldSet = explode(AddressFormat::FORMAT_NEW_LINE, AddressFormat::getAddressCountryFormat($idCountry));
```

Reply:
That's what I warned you about in the previous PR about dni, I wasn't sure either